File: tests/nodejs/cases/case_279_dead_sink_decoy_07.js
/**
 * Task case 279: dead_sink_decoy_07 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case279DeadSinkDecoy07(input) {
  const taskName = 'dead_sink_decoy_07';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case279DeadSinkDecoy07 };

File: tests/go/cases/case_279_dead_sink_decoy_07.go
package escape_tests

var retainedCase279 = []map[string]string{}

func Case279DeadSinkDecoy07(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_07",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_07:" + raw,
	}
	if 1 == 0 {
		retainedCase279 = append(retainedCase279, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}
